### DISCUSSION TITLE
Using Ketcher for drawing organic molecules at pre 16 organic chemistry level

### DISCUSSION THREAD

**Discussion by DorothyWarren** [epam/ketcher#1023] [Q&A]
I would like to use Ketcher for drawing some organic molecules required at secondary school level but have come across the following issues. I would be grateful if you can take a look at them and let me know if any of these are possible or if the system can be modified to allow them. Thanks in advance

1. I put together the reaction mechanism below but when I came to write in the text the letter 's' would not work - hence the word 'reaction' rather than 'step'! But when I closed and re-opened the programme the 's' worked. 
I also had a problem with saving this file - it would not save as a png or rnx 
![reaction mechanism](https://user-images.githubusercontent.com/94902446/143132289-cec6d6fd-7400-40fd-a5a2-0a7824f4414a.png)

2. On the equation below I want to show the reaction mechanism by drawing some curly arrows and delta+/- to show charge distribution. Is this possible?

![reaction](https://user-images.githubusercontent.com/94902446/143132415-89d7c252-bdb6-4deb-83c8-73557bf5abb3.png)

3. I need to draw a fully displayed formula, as written at GCSE i.e. with the central C showing. Is this possible? I would like to include the central C atoms -
![methane](https://user-images.githubusercontent.com/94902446/143132597-e19a484b-cda1-44ba-9dd2-c2058534ab0f.png)
 
4. Structural formula
The first bond length is always longer than the others. Can we change this?
When I substitute one of the Hs by a Cl - the C also goes and reverts to skeletal. - can we avoid this and keep all the atoms symbols showing
![image](https://user-images.githubusercontent.com/94902446/143133210-9923f47a-b628-4a1b-997f-4e270ef54539.png)

**Comment by AndreiMazol** [comment]
Dear @DorothyWarren ,

Thank you for interest to Ketcher.
Please provide more details about Ketcher version, operation system and browser version.

With reference to your examples:
1. I cannot reproduce the issue with 's' symbol so please provide more details (Ketcher version, operation system and browser version). The structure can be saved as reaction only  if one arrow has been added. Support of saving multiple arrows will be added in the future releases.
2. This moment curly arrows are not supported. It will be added in the future releases.
3. You need to change settings: settings-> atoms -> Display carbon explicitly (on)
![image](https://user-images.githubusercontent.com/72735611/143216951-c184ffac-1360-4e77-a6c1-de48ffc3decf.png)
4. It will be fixed. This moment you can change the lengths of bonds using selection tool. 
![image](https://user-images.githubusercontent.com/72735611/143217300-6a9f5221-d976-413e-a41d-0bce5d307cd7.png)

Looking forward for details from your side to proceed with improvements on Ketcher side.

Best regards,
Andrei

**Comment by DorothyWarren** [comment]
Dear Andrei,
Thank you for your swift response.
I.  I am using Ketcher v 2.3.1 on a windows 10 PC with a google chrome web
browser. The 's' does work with other drawings. It was only the example
that I showed you where it didn't work.Good to know that you will be
supporting multiple reactions in the future.
2. Pleased to hear that curly arrows will be used in future releases.
3. Thanks for showing me how to do C atoms - I have turned this on and it's
looking good.
4. For the bond lengths, I haven't managed to work this out - still showing
the first one much longer. PLease can you tell me exactly what the setting
should be. Sorry for being a little slow with this one.

[image: image.png]
Kind regards
Dorothy

**Comment by AndreiMazol** [comment]
Hi @DorothyWarren ,
please look  at video below to fix point 4

https://user-images.githubusercontent.com/72735611/143251066-2c8ac6c0-900b-4486-96b3-d1208e4e0892.mp4

**Comment by DorothyWarren** [comment]
Thank you so much for this video guide, it's really appreciated. All clear
now.
Best wishes
Dorothy.

**Comment by AndreiMazol** [comment]
Hi @DorothyWarren ,

I forgot to say about Ket formats: it allows to save all  visible objects from  canvas,  so I guess it should solve the issue highlighted in point num 1
![image](https://user-images.githubusercontent.com/72735611/143407036-7a7c9415-dd9c-4cf6-9343-56fe7040bc0a.png)
